webcompat/static/js/lib/ga.js

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+function getDNTStatus() {
+  // standard
+  if ("doNotTrack" in navigator) {
+    return navigator.doNotTrack;
+  }
+  // edge, ie11, safari
+  if ("doNotTrack" in window) {
+    return window.doNotTrack;
+  }
+  // ie 9 & 10
+  if ("msDoNotTrack" in navigator) {
+    return navigator.msDoNotTrack;
+  }
+
+  return "0";
+}
+
+if (getDNTStatus() !== "1") {
```

Comment:
Why testing different from 1 instead of testing equal to 0?

```js
   if (getDNTStatus() === "0") {…}
```

Reply:
Dunno, just the way my brain works, negative bias. :P

If they didn't say we can't track then, track them.

Reply:
(also, it's possible for the API to return "unspecified"... seems easier to just check for one thing)

https://developer.mozilla.org/en-US/docs/Web/API/Navigator/doNotTrack